src/main/resources/META-INF/rewrite/no-joda-time.yml

```diff
@@ -32,3 +32,6 @@ recipeList:
       version: 1.8.0
       onlyIfUsing: org.joda.time.*Interval*
   - org.openrewrite.java.joda.time.JodaTimeRecipe
+  - org.openrewrite.java.ChangeType:
```

Comment:
This is probably to less, I can remove this change for the moment when at my machine. Proper way would be to enable the type migration in the visitor and have a look than.